src/DotvvmAcademy.Web/Course/010_concepts/en/50_commands.md

```diff
@@ -10,19 +10,20 @@ CodeTask:
 
 # Commands
 
-__Commands__ are a kind of bindings used to invoke ViewModel's methods.
+The __Commands binding__ allows to call the ViewModel methods from the View.
```

Comment:
Typo in Command binding.

"Allows" is missing an object like "us". Alternatively you can use "is used".